src/frontend/youtube_clone/src/app/app.module.ts

```diff
@@ -1,16 +1,36 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
-
+import { MatSlideToggleModule } from '@angular/material/slide-toggle';
 import { AppRoutingModule } from './app-routing.module';
+
 import { AppComponent } from './app.component';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { MainPageComponent } from './main-page/main-page.component';
+import { VideoPageComponent } from './video-page/video-page.component';
+import { ProfilePageComponent } from './profile-page/profile-page.component';
+import { RouterModule } from '@angular/router';
+import {MatSidenavModule} from '@angular/material/sidenav';
```

Comment:
Code style
```suggestion
import { MatSidenavModule } from '@angular/material/sidenav';
```

Try to be consistent. I don't like to have two different code styles for import in the project

Reply:
will check before submitting from now onwards. 
Anyway i fixed it.